Added Launcher Icon and Attribution

yazh1202 opened the pull request.

avidraghav reviewed (changes requested):
Hi, thanks for the submitting the PR but please consider the following feedback:

1. It's not a good practice to use Image formats such as PNG for any drawables as they occupy alot of size. Always use vector drawables as they are light weight and can adapt to all resolutions.

2. It's good that you have added the PNG icons for all resolutions but let's not use PNG as they will increase the size of the app significantly. Try making an apk with all the PNG and you will see the difference yourself

3. So what you need to do is just add a vector drawable icon to the project. Preferably free to use, and if not add suitable attributions at the bottom of Readme.md file.

Just download a free icon as an SVG, convert it to a vector to drawable (Android Studio can do it for you) and apply the changes in PR. 🙂

yazh1202 commented:
Thanks for the feedback I will definitely be doing that for launcher icons from this point on.
I have added a new icon and removed the previous one,I have some doubts regarding where to place the svg files but Android Studio automatically stores them in drawables so I have left it there for only.

avidraghav commented:
> Thanks for the feedback I will definitely be doing that for launcher icons from this point on. I have added a new icon and removed the previous one,I have some doubts regarding where to place the svg files but Android Studio automatically stores them in drawables so I have left it there for now. Here is a screenshot:- ![IMG_20220322_223237](https://user-images.githubusercontent.com/84903954/159537503-d73318e0-7205-40a0-a918-52e0d5d865e3.jpg)

Thanks for the changes, Some old projects still use PNG for icons and the reason they quote is that Drawables can have performance issues as they are just lines of code. So It depends on project to project. My aim here is to not let the app size get big hence I asked you to use vector drawables.

Thanks for Impactful Contribution! I'll merge the PR now, don't forget to leave a 🌟 if MyNotes helped you in any way. Cheers!
